EED_Multi_Event_Registration.module.php

```diff
@@ -1172,15 +1173,16 @@ function( $additional_info ) {
 	 *    adjust_ticket_quantity
 	 *
 	 * @access    protected
-	 * @param EE_Ticket $ticket
-	 * @param int       $quantity
-	 * @param bool      $cart_update
+	 * @param EE_Ticket    $ticket
+	 * @param int          $quantity
+	 * @param EE_Line_Item $line_item
+	 * @param string       $action
 	 * @return bool
 	 */
-	protected function can_purchase_ticket_quantity( $ticket = null, $quantity = 1, $cart_update = false ) {
+	protected function can_purchase_ticket_quantity( $ticket = null, $quantity = 1, $line_item = null, $action = 'add' ) {
```

Comment:
this is older code so we were not type hinting for objects when this was written, but we do now, so could you please change the above line to:

```suggestion
	protected function can_purchase_ticket_quantity(
		EE_Ticket $ticket = null,
		$quantity = 1,
		EE_Line_Item $line_item = null,
		$action = 'add'
	) {
```